dotCMS/src/main/java/com/dotmarketing/business/PermissionBitAPIImpl.java

```diff
@@ -270,26 +270,31 @@ private boolean userPermissions(final UserProxy userProxy, final User userIn) {
 	public boolean doesUserHavePermission(final Permissionable permissionable, int permissionType, final User userIn, final boolean respectFrontendRoles) throws DotDataException {
 	    
 	    User user = (userIn==null || userIn.getUserId()==null) ? APILocator.getUserAPI().getAnonymousUser() : userIn;
-	    
-        if(user.getUserId().equals(APILocator.systemUser().getUserId())){
-            return true;
-        }
-        
-        if(user.isAdmin()) {
+
+
+	    if(user.getUserId().equals(APILocator.systemUser().getUserId())){
             return true;
         }
-        
-        if (permissionable == null) {
-            Logger.warn(this, "Permissionable object is null");
-            throw new NullPointerException("Permissionable object is null");
-        }
-        
-        if(UtilMethods.isEmpty(permissionable.getPermissionId())){
-            Logger.debug(this.getClass(), "Trying to get permissions on null inode of type :" + permissionable.getPermissionType()) ;
-            Logger.debug(this.getClass(), "Trying to get permissions on null inode of class :" + permissionable.getClass()) ;
-            return false;
-            
-        }
+
+		if (permissionable == null) {
+			Logger.warn(this, "Permissionable object is null");
+			throw new NullPointerException("Permissionable object is null");
+		}
+
+		if (UtilMethods.isEmpty(permissionable.getPermissionId())) {
+			Logger.debug(this.getClass(),
+					"Trying to get permissions on null inode of type :" + permissionable

```

Comment:
Remember to use Logger.debug(this.getClass(), ()-> "....") version

Reply:
done https://github.com/dotCMS/core/pull/20025/commits/4f24ed8150d8e42c574b1f93351d0d0e5813d9bc#diff-3680fcb1383b2f20c967e184d379c1c53a4fda303088b2c86573c83d6aa615daR286